mrp_partner_note/models/procurement_order.py

```diff
@@ -0,0 +1,17 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, api
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    @api.model
+    def _prepare_mo_vals(self, procurement):
+        res = super(ProcurementOrder, self)._prepare_mo_vals(procurement)
+        sale_proc = procurement.move_dest_id.procurement_id
+        res['notes'] = sale_proc.sale_line_id.order_id.partner_id.mrp_notes
```

Comment:
Don't overwrite this field if there's no value defined (maybe another module write this value)